tools/jdbc/src/test/java/org/duckdb/TestDuckDBJDBC.java

```diff
@@ -2150,6 +2150,14 @@ public static void test_connect_wrong_url_bug848() throws Exception {
         assertNull(d.connect("jdbc:h2:", null));
     }
 
+    public static void test_new_connection_wrong_url_bug10441() throws Exception {
+        try {
+            DuckDBConnection.newConnection("jdbc:duckdb@", false, new Properties());
```

Comment:
Can you use the assertThrows method here instead?

Reply:
With pleasure!

I saw the `try { …; fail(); } catch (…) { }` pattern used in some of the first few test methods in the class — is that something to tidy up in a separate PR?

Reply:
Yeah we need to go back and remove those old usages